src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -67,6 +69,21 @@ class WorkspaceMembersPage extends React.Component {
         this.hideConfirmModal = this.hideConfirmModal.bind(this);
     }
 
+    componentDidMount() {
+        const clientPolicyMembers = _.keys(this.props.policyMemberList);
+        Policy.openWorkspaceMembersPage(this.props.policy.id, clientPolicyMembers);
+    }
+
+    componentDidUpdate(prevProps) {
+        const isReconnecting = prevProps.network.isOffline && !this.props.network.isOffline;
+        if (!isReconnecting) {
+            return;
+        }
+
+        const clientPolicyMembers = _.keys(this.props.policyMemberList);
+        Policy.openWorkspaceMembersPage(this.props.policy.id, clientPolicyMembers);
```

Comment:
Same, use lodashGet, and I'd perform the falsey checking for the policyID here rather than in the Policy action.